Data Structures/Fenwick Tree/FenwickTree.java

```diff
@@ -0,0 +1,56 @@
+package Algos;
```

Comment:
1. Change to ```Data Structures/Trees/Fenwick Tree/java/fenwick_tree.java```
2. Create language folder``` java``` and place the file in it. Prefer ```snake_case``` for file names.